File: apps/native-ios/TechnoNexus/UI/Components/NexusCard.swift
import SwiftUI

struct NexusCard<Content: View>: View {
    var borderColor: Color = .white.opacity(0.1)
    let content: Content
    
    init(borderColor: Color = .white.opacity(0.1), @ViewBuilder content: () -> Content) {
        self.borderColor = borderColor
        self.content = content()
    }
    
    var body: some View {
        content
            .padding(24)
            .background(Color.glassWhite)
            .cornerRadius(32)
            .overlay(
                RoundedRectangle(cornerRadius: 32)
                    .stroke(borderColor, lineWidth: 1)
            )
    }
}

File: apps/native-android/app/src/main/java/com/technonexus/app/ui/components/NexusCard.kt
package com.technonexus.app.ui.components

import androidx.compose.foundation.background
import androidx.compose.foundation.border
import androidx.compose.foundation.layout.Box
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.clip
import androidx.compose.ui.graphics.Brush
import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp
import com.technonexus.app.ui.theme.GlassWhite

@Composable
fun NexusCard(
    modifier: Modifier = Modifier,
    borderColor: Color = Color.White.copy(alpha = 0.1f),
    content: @Composable () -> Unit
) {
    Box(
        modifier = modifier
            .clip(RoundedCornerShape(32.dp))
            .background(GlassWhite)
            .border(1.dp, borderColor, RoundedCornerShape(32.dp))
            .padding(24.dp)
    ) {
        content()
    }
}
